Formula/libspectre.rb

```diff
@@ -4,6 +4,7 @@ class Libspectre < Formula
   url "https://libspectre.freedesktop.org/releases/libspectre-0.2.9.tar.gz"
   sha256 "49ae9c52b5af81b405455c19fe24089d701761da2c45d22164a99576ceedfbed"
   license "GPL-2.0"
```

Comment:
Please change this to:
```
license "GPL-2.0-or-later"
```
From the `homepage`:

> libspectre is free software and is available to be redistributed and/or modified under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2, or (at your option) any later version.

Reply:
Done.